x/bank/client/cli/sendtx.go

```diff
@@ -47,16 +48,28 @@ func SendTxCmd(cdc *wire.Codec) *cobra.Command {
 
 			// build and sign the transaction, then broadcast to Tendermint
 			msg := client.BuildMsg(from, to, coins)
+
+			if viper.GetBool(flagAsync) {
+				res, err := ctx.EnsureSignBuildBroadcastAsync(ctx.FromAddressName, []sdk.Msg{msg}, cdc)
+				if err != nil {
+					return err
+				}
+				fmt.Println("Async tx send. tx hash: ", res.Hash)
```

Comment:
Should this be `Async tx sent`? (`sent` instead `send)

Reply:
meh